pkg/util/utils.go

```diff
@@ -118,10 +118,10 @@ type BugList map[string]BugResult
 type BugResult map[string][]Bug
 
 type Bug struct {
-	Summary      string `json:"name"`
-	ID           string
-	Url          string
-	FailureCount int32
+	Summary      string `json:"summary,omitempty"`
```

Comment:
this json field name value needs to stay as is, it has to match the json struct being returned by ci-search's json api:
https://search.ci.openshift.org/search?search=storage&maxAge=48h&context=-1&type=bug

(this struct is a bit overloaded, it's used both to deserialize the results coming from ci-search, as well as to storage bug metadata for reporting, but it saves mapping back and forth between two structs that otherwise serve basically the same purpose)